manufacturing/blower/README.md

```diff
@@ -68,14 +73,26 @@ We may have parts to send you from our inventory.
 [10rs]:   https://export.rsdelivers.com/product/jst/vhr-2n/jst-vhr-female-connector-housing-396mm-pitch-2/8201172
 [11amzn]: https://www.amazon.com/gp/product/B00N51OJJ4
 [12amzn]: https://www.amazon.com/gp/product/B011JC76OA
+[13rw]:   assets/blower_adapter.stl
+[14rw]:   assets/blower_adapter_holding_plate.dxf
+[15rw]:   assets/blower_driver_mounting.dxf
+[16rw]:   assets/blower_holding_plate.dxf
+
+
 
 ## Tools
 
 | Item | Quantity | Manufacturer  | Part #              | Price (USD) | Sources         | Notes |
 | ---- |---------:| ------------- | ------------------- | -----------:|-----------------| ----- |
 | a1   |        1 | Preciva       | PR-3254 (PR102001)  |    39.99 | [Z][a1amzn]     | Crimping tool for HX and HV connectors |
+| a2   |        1 | RespiraWorks  | Blower Adapter Gasket      | $1 (material) | [B][a2rw]            | Acrylic sheet cut from data in link |
+| a3   |        1 | RespiraWorks  | Blower Gasket      | $1 (material) | [B][a3rw]            | Acrylic sheet cut from data in link |
+
 
 [a1amzn]:https://www.amazon.com/gp/product/B07R1H3Z8X
+[a2rw]:assets/blower_adapter_gasket_.dxf
+[a3rw]:assets/blower_gasket_.dxf
```

Comment:
And again here is the DXF, but where are the other files? What is the logic for this?